go.mod

```diff
@@ -187,3 +187,5 @@ replace github.com/gogo/protobuf => github.com/gogo/protobuf v1.3.2
 go 1.19
 
 retract v0.0.2
+
+replace github.com/btcsuite/btcwallet v0.16.9 => github.com/yyforyongyu/btcwallet v0.16.8-0.20230509120640-2374a7fddef8
```

Comment:
hello, why not merge to btcsuite/btcwallet first?

Reply:
[this PR](https://github.com/btcsuite/btcwallet/pull/864) is under review. Once merged, we'll then update the mod here to merge this one.